js/app.js

```diff
@@ -0,0 +1,71 @@
+// Selectors for counters
+const TOTAL = 'total-zombies';
+const DEAD = 'dead-zombies';
+const SMALL = 'small-zombies';
+const MAD = 'mad-zombies';
+const STRONG = 'strong-zombies';
+
+let counters = {
+    [TOTAL]: 0,
+    [DEAD]: 0,
+    [SMALL]: 0,
+    [MAD]: 0,
+    [STRONG]: 0,
+};
+
+let calculateZombies = () => {
```

Comment:
1) const
2) what's the point of defining a function? it's just a one time call

please implement a solution without any function at all

Reply:
fixed